pyvista/core/filters.py

```diff
@@ -4326,6 +4326,42 @@ def strip(poly_data, join=False, max_length=1000, pass_cell_data=False,
         alg.Update()
         return _get_output(alg)
 
+    def shrink(poly_data, shrink_factor=1.0, inplace=False, progress_bar=False):
+        """Shrink a mesh
+
+        This helps shrink a mesh.
+
+        Parameters
+        ----------
+        poly_data : pyvista.PolyData
+            Mesh to shrink.
+
+        shrink_factor : float, optional
+            fraction of shrink for each cell.
+
+        inplace : bool, optional
+            Overwrites the original mesh inplace.
+
+        progress_bar : bool, optional
+            Display a progress bar to indicate progress.
+
+        Examples
+        --------
+        Extrude shrink mesh
+
+        >>> import pyvista
+        >>> mesh = pyvista.Sphere()
+        >>> mesh.shrink(shrink_factor=0.8).plot()
```

Comment:
```suggestion
        >>> shrunk_mesh = mesh.shrink(shrink_factor=0.8)
```

Let's not plot here.  It will cause tests to fail.

Reply:
> `shrunk_mesh`

@akaszynski always with the right semantics. I had to look this one up ([source](https://www.merriam-webster.com/words-at-play/shrink-shrank-shrunk-shrinked)).

Reply:
Thanks.